docs/burger-discounts/Discount.re

```diff
@@ -160,13 +160,34 @@ let getFreeBurger = (items: array(Item.t)) => {
     |> Js.Array.map(~f=Item.toPrice)
     |> Js.Array.sortInPlaceWith(~f=(x, y) => - compare(x, y));
 
-  switch (burgers[0], burgers[1]) {
-  | (Some(_), Some(cheaperPrice)) => Some(cheaperPrice)
-  | (None | Some(_), None | Some(_)) => None
+  switch (prices[1]) {
+  | None => None
+  | Some(cheaperPrice) => Some(cheaperPrice)
   };
 };
 // #endregion improved-get-free-burger
 
+ignore(getFreeBurger);
```

Comment:
Why is this necessary? To remove some "unused" warning I assume?

Reply:
Yeah, I have to use `ignore` quite a bit in source files whose code regions are imported into the text